lib/spack/spack/util/url.py

```diff
@@ -83,7 +83,7 @@ def is_path_instead_of_url(path_or_url):
         return not scheme
 
     # On Windows, we may have drive letters.
-    return "A" <= scheme <= "Z"
+    return "A" <= scheme.capitalize() <= "Z"
```

Comment:
Are the drive names `A` and `B` allowed? I thought they started at `C` for some historical reasons.

Reply:
They're legal drive names, historically they were reserved for floppy drives but I don't think that's technically enforced anymore.

Reply:
Okay, even if it is enforced, I don't think there is anything saying we can't install spack packages to a floppy.